tests/components/smlight/conftest.py

```diff
@@ -32,6 +47,18 @@ def mock_config_entry() -> MockConfigEntry:
     )
 
 
+@pytest.fixture
+def mock_config_entry2() -> MockConfigEntry:
```

Comment:
Can we rename the fixtures to be more meaningful?

Reply:
I have renamed this one `mock_config_entry_host`. I will make a followup PR to rename the existing fixture, and also use this new fixture in some of the config_entry tests.